### ISSUE TITLE
Tournament orders teams possibly wrong for KO mode

### ISSUE THREAD

**Issue by Debilski** [ASPP/pelita#664]
This result after round 1

```
Ranking after 10 matches (0 to go):
   The Smart Eating Players 8
   The Smart Random Players 5
       Smart Eating Players 4
    Random Explorer Players 2
                   Stopping 1
```

Gives us this ranking for the ko round:

```
 The Smart Eating Players ┐                       
                          ├─ ??? ┐                
 Smart Eating Players ────┘      │                
                                 ├─ ??? ┐         
 Random Explorer Players ─┐      │      │  ┏━━━━━┓
                          ├─ ??? ┘      ├──┨ ??? ┃
 The Smart Random Players ┘             │  ┗━━━━━┛
                                        │         
 Stopping ──────────────────────────────┘         
```

While it should be this?

```
 The Smart Eating Players ┐                       
                          ├─ ??? ┐                
 Random Explorer Players ─┘      │                
                                 ├─ ??? ┐         
 The Smart Random Players ┐      │      │  ┏━━━━━┓
                          ├─ ??? ┘      ├──┨ ??? ┃
 Smart Eating Players ────┘             │  ┗━━━━━┛
                                        │         
 Stopping ──────────────────────────────┘         
```

The idea would be that the 1st plays against the 4th and the 2nd against the 3rd. Or do we favour a different order? (1 against 3; 2 against 4)

Even if we keep the current system, the code still needs to be fixed, as it would just coincidentally give the correct result (we actually sort the list twice when going to ko mode …)

**Comment by otizonaizit** [comment]
The original idea was to have 1vs4, 2vs3, and then the last chance match. Not sure when it got lost.
So, yes, fix it!

**Comment by Debilski** [comment]
Fixed in #653.
